kong/core/balancer.lua

```diff
@@ -289,9 +293,50 @@ do
     end
   end
 
+  local creating = {}
```

Comment:
should we scope this and other `create_balancer` dependencies under a `do ... end` block? It'd be a little bit saner/easier to skim through imho

Reply:
Hmm I missed it; quite large for `do` block, but that's fine!